stuff/config/colornames.txt

```diff
@@ -18,27 +18,65 @@
   <color name="7">#FFFFFF</color>
   <!-- HTML Color Names -->
   <color name="AliceBlue">#F0F8FF</color>
+  <color name="Amber">#FFC000</color>
   <color name="AntiqueWhite">#FAEBD7</color>
+  <color name="AppleGreen">#83BC5D</color>
   <color name="Aqua">#00FFFF</color>
   <color name="Aquamarine">#7FFFD4</color>
+  <color name="Artichoke">#CAA435</color>
+  <color name="Avocado">#568203</color>
   <color name="Azure">#F0FFFF</color>
+  <color name="BabyBlue">#89CFEF</color>
+  <color name="Banana">#FCF4A3</color>
   <color name="Beige">#F5F5DC</color>
+  <color name="BeigeSienna">#D4948B</color>
+  <color name="Berry">#241571</color>
   <color name="Bisque">#FFE4C4</color>
   <color name="Black">#000000</color>
+  <color name="BlackCherry">#5F0133</color>
   <color name="BlanchedAlmond">#FFEBCD</color>
   <color name="Blue">#0000FF</color>
+  <color name="Blueberry">#4F86F7</color>
+  <color name="BlueLake">#4978C6</color>
+  <color name="BlueSlate">#8CC1EB</color>
   <color name="BlueViolet">#8A2BE2</color>
+  <color name="Blush">#DE5D83</color>
+  <color name="Bronze">#D2BAA2</color>
   <color name="Brown">#A52A2A</color>
+  <color name="BubbleGum">#FE5BAC</color>
+  <color name="Burgandy">#8D021F</color>
   <color name="BurlyWood">#DEB887</color>
+  <color name="BurntOchre">#B17551</color>
+  <color name="BurntOrange">#CC5500</color>
+  <color name="Byzantium">#702963</color>
   <color name="CadetBlue">#5F9EA0</color>
+  <color name="CadmiumOrange">#EA7E62</color>
```

Comment:
Randomly checking a color value...
Again, it's not clear where you are getting the values for these colors from.
CadmiumOrange (from google search):

> The color cadmium orange with hexadecimal color code #ed872d is a shade of orange. In the RGB color model #ed872d is comprised of 92.94% red, 52.94% green and 17.65% blue. In the HSL color space #ed872d has a hue of 28° (degrees), 84% saturation and 55% lightness.

When users enter a color name they will expect it to be an accurate and accepted value for that color.

The colornames feature does allow the user to override color values and assign their own but the defaults should align with standard color assignment.